### ISSUE TITLE
Error with graph and mdreport after upgrading to 0.4.7

### ISSUE THREAD

**Issue by vittominacori** [Consensys/surya#188]
Updating to 0.4.7 solve my issue with solidity parser but I have now other issues.

As you can see in this [commit](https://github.com/noncept/solidity-toolkit/commit/b1825db522a5b33d5169051baecfdc30be786572#diff-a3cbff837b34d6a811aca1c40bd1f6970b63a45732010a009fe9f35ac5f7afa0) it doesn't produce graph because I receive the following error `Error: <stdin>: syntax error in line 1 near '{'` and in mdreport it changed the file hash with `[object Promise]`.

To reproduce run 

`npx surya graph dist/SampleContract.dist.sol | dot -Tpng > analysis/control-flow/SampleContract.png`

`npx surya mdreport analysis/description-table/SampleContract.md dist/SampleContract.dist.sol`

**Comment by GNSPS** [comment]
I can't see a problem with the `mdreport` command, runs fine for me:

<img width="1640" alt="image" src="https://github.com/Consensys/surya/assets/4008213/32c042c4-43ca-4bad-a6aa-474101634bb3">

About the graph command, yes! Absolutely! I fucked up by leaving some variables being logged into the output for testing. Pushing a fix right now.

**Comment by vittominacori** [comment]
Yes. If I force use v0.4.6 it works but not for v0.4.7 or v0.4.8.

![image](https://github.com/Consensys/surya/assets/8151143/a13d7385-fd10-4ad9-a8f6-095e72be7c5a)

**Comment by GNSPS** [comment]
hmmm, I really don't know what's happening 🤷

**Comment by vittominacori** [comment]
Will try to understand. I have this issue in all my repos with 0.4.8.
